<code>File "kaggle_titanic_keras.py", line 3, in <module>
    from keras.models import Sequential
  File "/anaconda/lib/python2.7/site-packages/keras/__init__.py", line 4, in <module>
    from . import applications
  File "/anaconda/lib/python2.7/site-packages/keras/applications/__init__.py", line 1, in <module>
    from .vgg16 import VGG16
  File "/anaconda/lib/python2.7/site-packages/keras/applications/vgg16.py", line 14, in <module>
    from ..models import Model
  File "/anaconda/lib/python2.7/site-packages/keras/models.py", line 14, in <module>
    from . import layers as layer_module
  File "/anaconda/lib/python2.7/site-packages/keras/layers/__init__.py", line 4, in <module>
    from ..engine import Layer
  File "/anaconda/lib/python2.7/site-packages/keras/engine/__init__.py", line 8, in <module>
    from .training import Model
  File "/anaconda/lib/python2.7/site-packages/keras/engine/training.py", line 24, in <module>
    from .. import callbacks as cbks
  File "/anaconda/lib/python2.7/site-packages/keras/callbacks.py", line 25, in <module>
    from tensorflow.contrib.tensorboard.plugins import projector
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/__init__.py", line 30, in <module>
    from tensorflow.contrib import factorization
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/factorization/__init__.py", line 24, in <module>
    from tensorflow.contrib.factorization.python.ops.gmm import *
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/factorization/python/ops/gmm.py", line 27, in <module>
    from tensorflow.contrib.learn.python.learn.estimators import estimator
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/__init__.py", line 87, in <module>
    from tensorflow.contrib.learn.python.learn import *
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/__init__.py", line 23, in <module>
    from tensorflow.contrib.learn.python.learn import *
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/__init__.py", line 25, in <module>
    from tensorflow.contrib.learn.python.learn import estimators
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/estimators/__init__.py", line 297, in <module>
    from tensorflow.contrib.learn.python.learn.estimators.dnn import DNNClassifier
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/estimators/dnn.py", line 29, in <module>
    from tensorflow.contrib.learn.python.learn.estimators import dnn_linear_combined
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/estimators/dnn_linear_combined.py", line 31, in <module>
    from tensorflow.contrib.learn.python.learn.estimators import estimator
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/estimators/estimator.py", line 49, in <module>
    from tensorflow.contrib.learn.python.learn.learn_io import data_feeder
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/learn_io/__init__.py", line 21, in <module>
    from tensorflow.contrib.learn.python.learn.learn_io.dask_io import extract_dask_data
  File "/anaconda/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/learn_io/dask_io.py", line 26, in <module>
    import dask.dataframe as dd
  File "/anaconda/lib/python2.7/site-packages/dask/dataframe/__init__.py", line 3, in <module>
    from .core import (DataFrame, Series, Index, _Frame, map_partitions,
  File "/anaconda/lib/python2.7/site-packages/dask/dataframe/core.py", line 38, in <module>
    pd.computation.expressions.set_use_numexpr(False)
AttributeError: 'module' object has no attribute 'computation'
</code>